test/e2e/framework/kubectl/kubectl_utils.go

```diff
@@ -0,0 +1,212 @@
+/*
+Copyright 2022 The Koordinator Authors.
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
```

Comment:
We don't have to import some of these packages in `test/e2e/framework`, e,g,. `kubectl`, `kubelet`, and removing them could help us import less external dependencies in project.